### DISCUSSION TITLE
[Question] Directly downloading overlays from github

### DISCUSSION THREAD

**Discussion by starfern** [kubilus1/autoortho#431] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### question

Hello, 

I was wondering if I could directly download the overlays from github, since my download speed using AO is very slow and limited to 0.83MBps somehow (when I have 10gb internet speed limit).

Thanks in advance!

**Comment by Jeff2122** [comment]
@johnathanc929, I don't have a link for the zip files, but I know they're here somewhere. 
Are you having problems with the install, well one remedy is to revert to v0.6.0 or v0.6.1 to do the installation, then revert to v0.7.0 to get the repeated tiles fix.
Several people have complained that the latest version has problems with the download/install process.
